src/interface.rs

```diff
@@ -76,7 +77,15 @@ impl MenuMode {
             ResultSort::LastRun => menu_text.push_str("F1 - Switch Sort to Rank | "),
         }
 
-        menu_text.push_str("F2 - Delete");
+        menu_text.push_str("F2 - Delete | ");
+
+        match interface.result_filter {
+            ResultFilter::Global => menu_text.push_str("F3 - Show Directory Command History"),
```

Comment:
This text may be too long and overflow the width of many people's terminal windows. Maybe `F3 - This directory` and `F3 - All directories` or something?